test/open.js

```diff
@@ -2,6 +2,7 @@
 
 const spawk = require('spawk')
 const t = require('tap')
+const os = require('os')
```

Comment:
```suggestion
const os = require('node:os')
```

Reply:
We're trying to normalize on this pattern, changing it when we see it as part of any code we're touching, and making sure to use it in new cases.